src/common/summarize-profile.js

```diff
@@ -250,8 +271,10 @@ export function categorizeThreadSamples(profile) {
     });
 
     if (process.env.NODE_ENV === 'development') {
-      const uncategorized = countUncategorizedStacks(profile, summaries);
-      logUncategorizedSamples(uncategorized);
+      const cat = process.env.DUMP_CATEGORY || 'uncategorized';
```

Comment:
Oh, it's crap. I want this dumping frequently, for different categories, and this is a mostly useless way of getting it. It's useless in the web UI. I use it via my hacked-together method of running this stuff with the spidermonkey shell. Which is useless for anyone else.

I can remove the env var thing. I would like to keep the code for dumping out arbitrary categories, even if I have to access it by changing the hardcoded string here. Optimally, I would like a way to select a category on the summary page and download or view a dump of stacks, preferably with the reason they were categorized the way they were. I should learn me enough React to figure out how to do that, I guess.